app/controllers/patients_controller.rb

```diff
@@ -21,10 +21,12 @@ def index
 
   def new
     @patient = Patient.new
+    @patient_nav = false
     respond_with(@patient)
```

Comment:
Hmmm...so `respond_with()` is a wrapper around the `render` command.  In the render command, you are allowed to define "local" variables that are accessible to the view without necessarily adding them as member variables to the controller class. It's a bit cleaner.

In this case, I think you could remove all the `@patient_nav` variables and instead put this here:
  `respond_with(@patient, locals { show_nav: false })`

Then in the view below, you would do something like
```
- if local_assigns.fetch(:show_nav, true)
   / Code for the nav here
```
Normally you could interact with `shownav` as if it were a variable, but since we're testing for its presence, we reach around and directly talk to the `local_assigns` list.

See:

http://stackoverflow.com/questions/2060561/optional-local-variables-in-rails-partial-templates-how-do-i-get-out-of-the-de